.changeset/brown-camels-smash.md

```diff
@@ -0,0 +1,6 @@
+---
+"mobx-react": patch
```

Comment:
Let's do minor, as there might be some nuisance where people with old React typings need to update those as well (not fully sure).

Reply:
Ok, Sir.